deployment/chainloop/templates/_helpers.tpl

```diff
@@ -77,6 +77,12 @@ awsSecretManager:
   creds:
     accessKey: {{ required "access key required" .awsSecretManager.accessKey | quote }}
     secretKey: {{ required "secret key required" .awsSecretManager.secretKey | quote }}
+
+{{- else if eq .backend "gcpSecretManager" }}
```

Comment:
I believe this is missing a parent key, note how the other sections have `vault:` and `awsSecretManager:` in your case you are missing `gcp_secret_manager` or `gcpSecretManager` both would work I think.

note that I noticed this by rendering the chart before and after your change

1 - first I templated the chart with default settings and no changes and stored it in /tmp/before.yaml
2 - I applied your changes and updated a values.local.yaml file to point to gcpSecretManager and run the following command.
```
helm template test ./deployment/chainloop --values /tmp/values.local.yaml > /tmp/after.yaml && diff -u /tmp/before.yaml /tmp/after.yaml
```

```diff
 stringData:
   config.secret.yaml: |
     data:
@@ -128,22 +127,21 @@
         source: postgresql://chainloop:chainlooppwd@test-postgresql:5432/chainloop-cp
 
     credentials_service:      
-      vault:
         secretPrefix: "chainloop"
-        address: "http://test-vault:8200"
-        token: "notapassword"
+        projectId: "projectID"
+        authKey: "authkey"
```

and notice how the config is missing a namespace key.